OrbitClientGgp/CMakeLists.txt

```diff
@@ -7,19 +7,20 @@ project(OrbitClientGgp)
 add_executable(OrbitClientGgp main.cpp)
 
 target_compile_options(OrbitClientGgp PRIVATE
-    ${STRICT_COMPILE_FLAGS})
+        ${STRICT_COMPILE_FLAGS})
 
-target_include_directories(OrbitClientGgp PRIVATE 
-    ${CMAKE_CURRENT_LIST_DIR})
+target_include_directories(OrbitClientGgp PRIVATE
+        ${CMAKE_CURRENT_LIST_DIR})
 
 target_sources(OrbitClientGgp PRIVATE
-    ClientGgp.cpp
-    ClientGgp.h
-    main.cpp)
+        ClientGgp.cpp
+        ClientGgp.h
+        main.cpp)
 
-target_link_libraries(OrbitClientGgp PUBLIC 
-    OrbitBase
-    OrbitCaptureClient
-    OrbitClientServices
-    OrbitProtos)
+target_link_libraries(OrbitClientGgp PUBLIC
+        OrbitBase
+        OrbitCaptureClient
+        OrbitClientServices
+        OrbitGl
```

Comment:
OrbitClientGgp must be independent from OrbitGl so please do not include it, thanks!

Reply:
I understand. Thank you!